tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -761,7 +761,10 @@ impl<'a> field::Visit for DefaultVisitor<'a> {
 
     fn record_error(&mut self, field: &Field, value: &(dyn std::error::Error + 'static)) {
         if let Some(source) = value.source() {
-            self.record_debug(field, &format_args!("{}, {}: {}", value, field, source))
+            self.record_debug(
+                field,
+                &format_args!("{}, {}.sources: {}", value, field, ErrorSourceList(source)),
```

Comment:
the default formatter will separate fields with a space (no comma) and delimits keys and values with an `=` rather than a `: `. this looks like the same formatting as the `pretty` formatter. we should change this so it matches the rest of the fields formatted by this formatter:
```suggestion
                &format_args!("{} {}.sources={}", value, field, ErrorSourceList(source)),
```

(otherwise, we might output things like
```
foo=bar err="something bad", err.sources: [something bad, something else] baz=10
```
which doesn't look great)

Reply:
...oh, i see that you already fixed this in https://github.com/tokio-rs/tracing/pull/1460/commits/8e0c804945028752481afc1497dba9fcdd8caf3b while i was writing the review comment, never mind! :)

Reply:
:)